css/index.css

```
@@ -129,3 +133,109 @@ footer {
   width: 60px;
   height: 60px;
 }
+
+#profile_picture {
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+}
+
+/* Add a navigational header to your webpage */
+/* The navigation menu should include a link to each of your sections (i.e. "About", "Experience", etc.) */
+/* Make the header sticky/fixed on the page */
+
+header {
+  background-color: #f1e8e8;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.navBar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding: 1rem;
+}
+.menuItem {
+  font-size: 1.125rem;
+  padding: 1rem;
+}
+
+/* Change the layout of your "Experience" section using rows and columns so items are displayed in a grid rather than a list */
+.experience-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 20px;
+}
+
+.experience-item {
+  background-color: #f0f0f0;
+  padding: 20px;
+  text-align: left;
+}
+
+/* Update the "Leave a Message" form to use Flexbox or CSS Grid */
+.message-form {
+  display: flex;
+  flex-direction: column;
+  max-width: 400px;
+  margin: 0 auto;
+}
+.form-group {
+  margin-bottom: 10px;
+  padding: 8px;
+}
+
+.message-form button[type="submit"] {
+  background-color: #333;
+  color: #fff;
+  padding: 10px 15px;
+  border: none;
+  cursor: pointer;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+  color: #574d4d;
+}
+
+#profileName {
+  font-size: 1.5rem;
+  color: #574d4d;
+}
+
+/* Use Media Queries to make sure the webpage is responsive for all device sizes (mobile, tablet, desktop, etc.) */
+/* Media query for mobile devices */
+@media (max-width: 767px) {
+  header a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
+  header-left {
+    float: left;
+  }
+  .experience-container {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 20px;
+  }
+}
+
+/* Media query for tablet devices */
+@media (min-width: 768px) and (max-width: 1023px) {
+}
```

If there's nothing here, either delete:
```suggestion
```

Or make it clear that you will come back to this:
```suggestion
/* Media query for tablet devices */
@media (min-width: 768px) and (max-width: 1023px) {
  /* TODO */
}
```